src/app/helpers/telemetryHelper.js

```diff
@@ -131,17 +131,17 @@ module.exports = {
   /**
    * This function helps to get params data for log event
    */
-  getParamsData: function (options, statusCode, resp, uri) {
+  getParamsData: function (options, statusCode, resp, uri, traceid) {
     const apiConfig = telemtryEventConfig.URL[uri]
     let params = [
       { 'title': apiConfig && apiConfig.title },
       { 'category': apiConfig && apiConfig.category },
-      { 'url': apiConfig && apiConfig.url },
+      { 'url': options.path || apiConfig && apiConfig.url },
       { 'duration': Date.now() - new Date(options.headers.ts) },
       { 'status': statusCode },
       { 'protocol': 'https' },
       { 'method': options.method },
-      { 'req': options.body }
+      { 'traceid': traceid }
```

Comment:
is req not required here ?
why did you remove ?

Reply:
this method is for entry log , in the previous review vinu said , we no need to pass the req body in this method.so I removed it here